core/rfb.js

```diff
@@ -967,13 +960,13 @@ RFB.prototype = {
         if (!this._rfb_credentials.username ||
             !this._rfb_credentials.password ||
             !this._rfb_credentials.target) {
-            var event = new CustomEvent("credentialsrequired",
-                                        { detail: { types: ["username", "password", "target"] } });
-            this.dispatchEvent(event);
+            this.dispatchEvent(new CustomEvent(
+                "credentialsrequired",
```

Comment:
These line breaks seem unnecessary. Are the lines really that long?

Reply:
I don't think we need a consistent line-break pattern for each dispatchEvent call. It's fine if the line-breaks are on a case-to-case basis.

Reply:
Most dispatchEvents are the same length.
We just had very long lines and not-so-long ones.

I introduce the change because some of you guys asked me to put the CustomEvent inside of the dispatchEvent in all cases (before it was mixed) and that made most dispatch to be on the long size.

I think that it looks cleaner and better as it is now tbh.
Would you rather me to rever all of them?